packages/jest-circus-allure-reporter/src/step-wrapper.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable require-jsdoc */
```

Comment:
In the interest of getting this code reviewed sooner I have disabled the `require-jsdoc` rule for this file.